recipes/bcftools/1.5/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version="1.5" %}
+
+package:
+  name: bcftools
+  version: {{ version }}
+
+build:
+  number: 4
+
+source:
+  url: https://github.com/samtools/bcftools/releases/download/{{ version }}/bcftools-{{ version }}.tar.bz2
+  sha256: 9ff75a680891467776d7064f41deb6be7a595030ba807494727cd5bee7842301
```

It would be good to include this patch from the the previous 1.5 build 3:
```suggestion
  sha256: 9ff75a680891467776d7064f41deb6be7a595030ba807494727cd5bee7842301
  patches:
    - b37b03e815cd4f25af70a57c08449f8c719dcc6a.patch # from https://github.com/samtools/bcftools/pull/645

```